### ISSUE TITLE
DI.FM, Rock Radio, Jazz Radio, Radio Tunes, Classical Radio, Zen Radio - scrobbling doesn't work.

### ISSUE THREAD

**Issue by Rostov84** [web-scrobbler/web-scrobbler#3579]
**Describe the bug**

Scrobble on the following sites has not been working for several days:

- https://www.di.fm/
- https://www.rockradio.com/
- https://www.jazzradio.com/
- https://www.classicalradio.com/
- https://www.radiotunes.com/
- https://www.zenradio.com/

After accessing the websites of the above-mentioned radio stations, the Web Scrobbler icon turns red, which means that scrobbling from this website is possible. However, when music is turned on, scrobbling doesn't occur. Scrobble works on other websites.

**Environment (please complete the following information)**
-   OS: Windows 7 Home Premium SP1 (x64)
-   Browser: Google Chrome  109.0.5414.120 (x64)
-   Extension version: 2.86.1

**Comment by Maddieman** [comment]
I had this issue with rockradio.com yesterday, but not di.fm.  Both seem to be working for me today (in chrome, macos).

**Comment by Rostov84** [comment]
Yes, I confirm that scrobbling works for me today on all the sites I mentioned in the previous post. Weird because it was down for two days and there has been no update to Web Scrobbler since version 2.86.1.

**Comment by rob9315** [comment]
all of them seem to work
